crates/datasources/src/clickhouse/mod.rs

```diff
@@ -361,3 +454,40 @@ impl fmt::Debug for ClickhouseExec {
             .finish_non_exhaustive()
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct ClickhouseTableRef<'a> {
+    schema: Option<Cow<'a, str>>,
```

Comment:
I'd make this field `database` so that it's consistent with clickhouse terminology (and we just have a comment saying a database in clickhouse is similar to a schema elsewhere).